### DISCUSSION TITLE
怎么修改字段类型啊

### DISCUSSION THREAD

**Discussion by Hby118** [apache/doris#20981] [RFC]
ALTER TABLE db.sss MODIFY STRING varchar(255)

原来是date修改成varchar255

但是报错

1.1.3版本

**Comment by yagagagaga** [comment]
Here is the [document](https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-TABLE-COLUMN) and your Doris version is too old, I recommend your upgrade to latest.

**Reply by Hby118** [reply]
但是我前一阵子改过一次啊而且成功了,当时也是各种百度搜的,又因为电脑死机过 sql都没保存所以忘了,现在又查不到了.

之前一次是把一个字段的varchar(255) 改成了varchar(5000),成功了的

**Reply by yagagagaga** [reply]
The official documents provide a large number of examples that you can refer to.
https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-TABLE-COLUMN

**Reply by Hby118** [reply]
谢谢,看了条重要信息,所以报错.

除DATE与DATETIME以外都可以转换成STRING，但是STRING不能转换任何其他类型
